synology_dsm/api/surveillance_station/__init__.py

```diff
@@ -0,0 +1,122 @@
+"""Synology SurveillanceStation API wrapper."""
+import urllib
+
+from .camera import SynoCamera
+from .const import MOTION_DETECTION_BY_SURVEILLANCE, MOTION_DETECTION_DISABLED
+
+
+class SynoSurveillanceStation(object):
+    """An implementation of a Synology SurveillanceStation."""
+
+    API_KEY = "SYNO.SurveillanceStation.*"
+    CAMERA_API_KEY = "SYNO.SurveillanceStation.Camera"
+    CAMERA_EVENT_API_KEY = "SYNO.SurveillanceStation.Camera.Event"
+    HOME_MODE_API_KEY = "SYNO.SurveillanceStation.HomeMode"
+    SNAPSHOT_API_KEY = "SYNO.SurveillanceStation.SnapShot"
+
+    def __init__(self, dsm):
+        """Initialize a Surveillance Station."""
+        self._dsm = dsm
+        self._cameras_by_id = {}
+
+    def update(self):
+        """Update cameras and motion settings with latest from API."""
+        response = self._dsm.get(self.CAMERA_API_KEY, "list")
+        for data in response["data"]["cameras"]:
+            if data["id"] in self._cameras_by_id:
+                self._cameras_by_id[data["id"]].update(data)
+            else:
+                self._cameras_by_id[data["id"]] = SynoCamera(data)
+
+        for camera_id in self._cameras_by_id:
+            self._cameras_by_id[camera_id].update_motion_detection(
+                self._dsm.get(
+                    self.CAMERA_EVENT_API_KEY, "MotionEnum", {"camId": camera_id}
+                )["data"]
+            )
+
+        live_view_response = self._dsm.get(
+            self.CAMERA_API_KEY,
+            "getLiveViewPath",
+            {"idList", self._cameras_by_id.keys()},
```

Comment:
```suggestion
            {"idList": ",".join(str(k) for k in self._cameras_by_id.keys())},
```
Not sure if there is a better way to do it but this works.

Reply:
Oh yeah, indeed ! :sweat_smile: 

`",".join(self._cameras_by_id.keys())` isn't enought ?

Reply:
keys are int but join expects a sequence of strings.